### DISCUSSION TITLE
Cache or download

### DISCUSSION THREAD

**Discussion by ALMIGHTY-VC** [fast4x/RiMusic#2990] [Q&A] [answered]
As a conservative net user i thought while listening on the go i can save my songs for offline playback (i.e caching) 

But i just red this
[https://github.com/fast4x/RiMusic/discussions/92#discussioncomment-7665203]
And thought am i sacrificing my performance?? 

Summary:-
Does cached songs slow down the app performance and what are the pros and cons of downloading in front of caching there isnt one in quality of song is there as i always listen in high quality

**Comment by fast4x** [comment, marked as answer]
Caching improve player performance because it not to download always. 
Quality is related to option in settings, if you set low it will cache in low quality, so set to automatic and go ready for best quality available.

**Reply by ALMIGHTY-VC** [reply]
Thanks will continue caching🙌🙏
